src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -193,7 +202,6 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
         string calldata socket,
         OperatorKickParam[] calldata operatorKickParams
     ) external {
-        require(quorumNumbers.length == operatorKickParams.length, "BLSIndexRegistryCoordinator.registerOperatorWithCoordinator: quorumNumbers and operatorKickParams must be the same length");
```

Comment:
is there a reason we no longer need this check? feels like a good sanity-check to me.
relatedly, do we have adequate checks on the `quorumNumbers` input in general? For example, do we need to check that it contains no duplicates?

Reply:
its actually not the case!

the operator can register for more quorums than are at the cap. in this case they only kick operators out of the quorums that have reached the cap.

as for the other checks on `quorumNumbers`, this should be handled in the internal call to `_registerOperatorWithCoordinator`

Reply:
OK, that makes sense but IMO the documentation here is lacking, particularly w/r/t requirements/explanation of `operatorKickParams`.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/88/commits/c7204044c3cec47c02f205d069c2f29246496ab4 70e0c7da